connect/src/main/java/com/stripe/android/connect/EmbeddedComponentManager.kt

```diff
@@ -105,4 +178,50 @@ class EmbeddedComponentManager(
     data class Configuration(
         val publishableKey: String,
     ) : Parcelable
+
+    companion object {
+        @VisibleForTesting
+        internal val permissionsFlow: MutableSharedFlow<Boolean> = MutableSharedFlow(extraBufferCapacity = 1)
+        private val launcherMap = mutableMapOf<Activity, ActivityResultLauncher<String>>()
+
+        /**
+         * Hooks the [EmbeddedComponentManager] into this activity's lifecycle.
+         *
+         * Must be called in [ComponentActivity.onCreate], passing in the instance of the
+         * activity as [activity]. This must be called in all activities where an EmbeddedComponent
+         * view is used.
+         */
+        fun onActivityCreate(activity: ComponentActivity) {
+            val application = activity.application
+
+            application.registerActivityLifecycleCallbacks(object : ActivityLifecycleCallbacks {
```

Comment:
Why register on the Application? I think it was better before to register on the Activity. Reading [the docs](https://developer.android.com/reference/android/app/Activity#registerActivityLifecycleCallbacks(android.app.Application.ActivityLifecycleCallbacks)), it turns out you actually don't need to unregister the callback:
> As this callback is associated with only this Activity, it is not usually necessary to [unregister](https://developer.android.com/reference/android/app/Activity#unregisterActivityLifecycleCallbacks(android.app.Application.ActivityLifecycleCallbacks)) it unless you specifically do not want to receive further lifecycle callbacks.

Reply:
I also prefer the callbacks on the activity, but that API is only available on API 29+ (whereas the Application version is available on 14+): https://developer.android.com/reference/android/app/Activity#registerActivityLifecycleCallbacks(android.app.Application.ActivityLifecycleCallbacks)